hirosystems/bitcoin-core/templates/statefulset.yaml

```diff
@@ -171,6 +171,9 @@ spec:
               containerPort: {{ .Values.containerPorts.rpc }}
             - name: tcp-p2p
               containerPort: {{ .Values.containerPorts.p2p }}
+            { { - if .Values.extraPorts } }
+            { { - include "common.tplvalues.render" (dict "value" .Values.extraPorts "context" $) | nindent 12 } }
+            { { - end } }
```

Comment:
Can we make sure the formatting for the braces matches the formatting elsewhere?

Reply:
not sure why my copy/paste does this.  its fixed now